### DISCUSSION TITLE
Are serverless functions down ?

### DISCUSSION THREAD

**Discussion by carlosdarwed** [vercel/community#2847] [Help] [answered]
### Page to Investigate

Serverless functions

### Steps to Reproduce

Service written in Next JS isn't being able to connect again, it is deployed as a function on Vercel, it went down aproximately 3 minutes ago

**Comment by swarnava** [comment, marked as answer]
Yes we are experiencing an outage: https://www.vercel-status.com/incidents/cqkdqrzqkspm

**Reply by lautapercuspain** [reply]
We're getting  _- Gateway Timeout (Vercel) -_ Our site is offline.

**Reply by swarnava** [reply]
Can you share the link?

**Reply by LeonAlvarez** [reply]
Same here site down, cant build. Asume that the issue is build happens on the affected instance
<img width="899" alt="image" src="https://github.com/vercel/community/assets/22636643/121e8437-e1a1-4c7b-b563-f30deec174c2">

**Reply by swarnava** [reply]
This incident is only affecting to `iad1` region. If you change Serverless Function `region` to `cle1` (Cleaveland) or another region, it should work.

**Reply by LeonAlvarez** [reply]
Fixed, seems that after update redeploy took old settings, I did another deploy from CLI and got solved. Thanks
